### ISSUE TITLE
Sponsorship

### ISSUE THREAD

**Issue by therebelrobot** [Utah-Node-Ninjas/unn-committee#4]
We need a better plan going forward for obtaining and communicating with sponsors.

Some stuff we may need:
- [x] extension cord
- [ ] surge protector
- [ ] chromecast

**Comment by ghost** [comment]
I'm bringing a 14' extension cord which has 3 grounded outlets.

**Comment by ghost** [comment]
Contacted the following sponsors:
-  [x] O.C. Tanner, regarding hosting in SLC, awaiting response

**Comment by ghost** [comment]
I submitted request here:
https://www.jetbrains.com/devnet/sponsorship/usergroups/

**Comment by therebelrobot** [comment]
Talk to Derek about jetbrains, I think he had some contact with them.

**Comment by ghost** [comment]
@derekargueta , Did you already contact jetbrains?

**Comment by ghost** [comment]
We have an offer of sponsorship from Overstock.com for the SLC chapter.  I haven't heard anything from O.C.Tanner.  @joeycozza and @ryanswapp , any objections to going with Overstock?

**Comment by joeycozza** [comment]
No objections here

**Comment by ryanswapp** [comment]
Sounds good to me.

**Comment by ghost** [comment]
Details with Overstock are still being ironed out, but It looks like it might be only once a month.  I might hold out and continue looking for sponsor to get a twice a month schedule in one location, if no one objects.

**Comment by ryanswapp** [comment]
I don't have a problem with either!

**Comment by ghost** [comment]
I'm closing old issues that are no longer relevant.
